core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -26,7 +26,7 @@ import com.codahale.metrics.{Gauge, MetricRegistry}
 
 import org.apache.spark.scheduler._
 import org.apache.spark.metrics.source.Source
-import org.apache.spark.util.{ThreadUtils, Clock, SystemClock, Utils}
+import org.apache.spark.util.{MonotonicClock, ThreadUtils, Clock}
```

Comment:
that's interesting: clearly IntelliJ idea makes ordering decisions within an import line than between them. Fixed to alpha order —and something to look out for in future.